### ISSUE TITLE
[Bug]: Neutronium bee not working in Apiary MK II

### ISSUE THREAD

**Issue by phrostt** [AllTheMods/ATM-9#2045]
### Possible Fixes

Yes

### Modpack Version

9-0.3.2

### What happened?

Apiary MK II works for all the bees tested except for Neutronium. Neutronium block for flower, neutronium bee in cage with very high productivity, weather tolerance: any, and metaturnal. The machine does not start working.

**Comment by Pumpkin7266** [comment]
@loving2 i think you fixed this one right?

**Comment by loving2** [comment]
Yep, the name requirement on bees has been removed from the Apiary entirely, so this should be fixed on ATM9 v1.0.3.
